docs/stable/2-account.md

```diff
@@ -81,19 +80,23 @@ Everything else associated with the contact is either verified or derived from t
     - Timestamp
 - These bundles are made available in a variety of ways, as defined in section 2.1.
 
+```js
+// TODO is there an "Open Waku Systems" ^^^?
```

Comment:
Open Whisper Systems has nothing to do with the Whisper protocol.

Reply:
Ah, thank you. I'll add a reference to https://en.wikipedia.org/wiki/Signal_Messenger#2013%E2%80%932018:_Open_Whisper_Systems